packages/flutter/lib/src/foundation/unicode.dart

```diff
@@ -19,19 +16,19 @@ class Unicode {
   /// Treat the following text as embedded left-to-right.
   ///
   /// Use [PDF] to end the embedding.
-  static const String LRE = '\u202A';
+  static const int LRE = 0x202A;
```

Comment:
These changes from string to int seem to make more code longer than shorter. What's the motivation for them?